WIP Update Readme and add docs (#1)
Kava-Labs/switch: README.md

<p align="center"><img src="./src/assets/switch-logo.svg" width="300" /></p>
<p align="center">
💸 Swap BTC, ETH, XRP in seconds. Keep your private keys private. 🔒
</p>

> ![Switch swap screen cropped](./screenshots/swap-eth-btc-cropped-faded.png)

Reviewer: Also, the "hero" screenshot looks really good!

[![GitHub All Releases](https://img.shields.io/github/downloads/kava-labs/switch/total.svg)](https://github.com/Kava-Labs/switch/releases)
[![GitHub](https://img.shields.io/github/license/Kava-Labs/switch.svg)](https://github.com/Kava-Labs/switch/blob/master/LICENSE)
[![Discourse users](https://img.shields.io/discourse/https/forum.interledger.org/users.svg)](https://forum.interledger.org)
[![Twitter Follow](https://img.shields.io/twitter/follow/kava_labs.svg?label=Follow&style=social)](https://twitter.com/kava_labs)
</div>

Switch is the fastest, most secure way to swap cryptocurrencies. It uses the power of streaming payments and payment channels to get lightning fast swaps, enables interoperability across all blockchains, all while preserving self-custody.

Load funds onto "cards", easily swap between them, and unload when you're done. Only you have access to these funds and, thanks to streaming micropayments, even while trading you retain full asset custody!

# Get Started

 1) Download the latest testnet release for Mac, Windows, and Linux [here](https://github.com/Kava-Labs/switch/releases/latest).

>     - To open Switch on mac, you'll need to right click and select open.

Reviewer: ```suggestion
    - To open Switch on Mac, right click or Control click on the app icon, and select "Open."
```

 2) Add a minimum of two cards. We suggest ETH and XRP to get started.

>     - Add an Ethereum card using a private key. If you don't have a kovan account, get a private key and address from [this generator](https://vanity-eth.tk). Then load the address with testnet eth at [the Kovan testnet faucet](https://faucet.kovan.network/).

Reviewer: ```suggestion
    - Add an Ethereum card using a private key. If you don't have a Kovan testnet account, generate a private key and address [here](https://vanity-eth.tk). Then, load the address with testnet ether at [the Kovan testnet faucet](https://faucet.kovan.network/).
```

>     - Add an XRP card using a xrp secret. If you don't have a test xrp account, get a prefunded testnet secret from [the XRP testnet faucet](https://developers.ripple.com/xrp-test-net-faucet.html).

Reviewer: ```suggestion
    - Add an XRP card using a XRP secret. If you don't have a testnet account, generate a prefunded testnet account and secret from [the XRP testnet faucet](https://developers.ripple.com/xrp-test-net-faucet.html).
```

<p align="middle">
  <img src="./screenshots/home-eth-btc-xrp.png" width="400" />
  <img src="./screenshots/swap-xrp-eth-success.png" width="400" />
</p>

# How It Works
## Fast Non-custodial Trading

When trading between assets, Switch will first send a very small amount of the source asset, the equivalent of $0.05, to the exchange party known as the connector. Then, the connector sends some amount of the destination asset. If the connector upholds its side of the bargain and the exchange rate it provides is decent, we repeat the process. And again. And again; many times per second.

This is the model of streaming micropayments: moving value bit-by-bit until the entire payment or trade is complete.

If at any point the connector stops sending or sends too little of the destination asset, we halt the exchange. This enables non-custodial trading, since the counterparty risk is merely a few cents.

Payment channels enable these payments to be cheap and fast.

Switch uses Kava as the default connector, but we will be expanding this to user-defined connectors in the near future.

## Links
 - [FAQs](./docs/faqs.md)
 - [Setup Swaps Over Lightning](./docs/lightning-setup.md)

>  - [What is layer 2 and payment channels?](./docs/faqs.md#what-is-layer-2-and-payment-channels)

Reviewer: ```suggestion
 - [What is layer 2? What are payment channels?](./docs/faqs.md#what-is-layer-2-and-payment-channels)
```

>  - [What is interledger?](./docs/faqs.md#what-is-interledger)

Reviewer: ```suggestion
 - [What is Interledger?](./docs/faqs.md#what-is-interledger)
```
